From 8d36c877d2528cb8634219949730767b0e6e5150 Mon Sep 17 00:00:00 2001
From: Kent Fredric <kentnl@gentoo.org>
Date: Fri, 24 Apr 2020 03:44:51 +1200
Subject: Remove sharedir nuking logic.

This breaks install on Gentoo, as it tries to remove the *system*
share_dir *before* Gentoo gets around to things.

And this code is basically redundant because Gentoo assurces cruft
entries in share will get removed, which is not something people using
CPAN clients can expect.

https://bugs.gentoo.org/719074
---
 inc/My/Builder.pm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/inc/My/Builder.pm b/inc/My/Builder.pm
index 6f5a97e..f101beb 100644
--- a/inc/My/Builder.pm
+++ b/inc/My/Builder.pm
@@ -47,7 +47,7 @@ sub ACTION_install
 
  $sharedir = eval {File::ShareDir::dist_dir('Alien-SDL')} || '';
 
- if ( -d $sharedir )
+ if ( 0 )
  {
    print "Removing the old $sharedir \n";
    remove_tree($sharedir);
-- 
2.26.2

